foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/ConfigLoader.java

```diff
@@ -47,6 +47,7 @@
 public class ConfigLoader {
     private static final Logger logger = LoggerFactory.getLogger(ConfigLoader.class);
     private PropertyDao propertyDao;
+    public boolean fullyLoaded;
```

Comment:
Think we don't actually need this attribute anymore?

Reply:
It's removed